pyphare/pyphare/pharein/simulation.py

```diff
@@ -681,10 +703,15 @@ def within_simulation_extent(self, extent):
 
 
 
+    def format_restart_time(self, time):
+        # five digits on each side of decimal point, 1 for decimal point
```

Comment:
this could probably be done in C++ and provided via pybind so there's only one real impl